client/src/components/Footer/__tests__/Footer.test.tsx

```diff
@@ -1,18 +1,27 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import Footer from '..';
-
-test('renders Inferno Footer', () => {
-  render(<Footer />);
-  const linkElement = screen.getByText(/Open Source/i);
-  expect(linkElement).toBeInTheDocument();
-});
-
-test('should navigate elsewhere when link is clicked', () => {
-  render(<Footer githubLink="https://github.com/onc-healthit/inferno" />);

```

Comment:
Do we actually use the Footer in the interface?  Ok if not (I understand out of scope of this update), but I was trying to figure out where this shows up and couldn't figure it out ;-)

Reply:
Footer is completely unused, not sure if we should add it in? It doesn't really serve much purpose at the moment, though.